packages/medusa/src/api/routes/admin/products/index.ts

```diff
@@ -116,7 +118,7 @@ export const allowedFields = [
   "metadata",
 ]
 
-export const allowedRelations = [
+export const allowedAdminProductRelations = [
   "variants",
   "variants.prices",
   "images",
```

Comment:
Should `filterableFields also be `filterableAdminProductFields` to avoid conflicts when exporting everything

Reply:
Ah I actually don't think, this is used :) Thanks